frontend/src/pages/_document.tsx

```diff
@@ -0,0 +1,79 @@
+import RawDocument, {
+  DocumentContext,
+  Head,
+  Html,
+  Main,
+  NextScript,
+} from "next/document";
+import React from "react";
+import { ServerStyleSheet } from "styled-components";
+
+export default class Document extends RawDocument {
+  static async getInitialProps(context: DocumentContext) {
+    const sheet = new ServerStyleSheet();
+    const originalRenderPage = context.renderPage;
+
+    try {
+      context.renderPage = () =>
+        originalRenderPage({
+          enhanceApp: (App) => (props) =>
+            sheet.collectStyles(<App {...props} />),
+        });
+
+      const initialProps = await RawDocument.getInitialProps(context);
+
+      return {
+        ...initialProps,
+        styles: (
+          <>
+            {initialProps.styles}
+            {sheet.getStyleElement()}
+          </>
+        ),
+      };
+    } finally {
+      sheet.seal();
+    }
+  }
+
+  render() {
+    return (
+      <Html>
+        <Head>
+          <link
+            rel="icon"
+            type="image/png"
+            sizes="32x32"
+            href="/favicon-32x32.png"
+          />
+          <link
+            rel="icon"
+            type="image/png"
+            sizes="16x16"
+            href="/favicon-16x16.png"
+          />
+          <link
+            rel="apple-touch-icon"
+            sizes="180x180"
+            href="/apple-touch-icon.png"
+          />
+          <link
+            rel="manifest"
+            href="/site.webmanifest"
+            crossOrigin="use-credentials"
```

Comment:
We need `crossOrigin` here in order to send cookies along with the request to prevent redev from blocking and redirecting the request to Okta. Thanks to @jgadling for finding this solution 🎉 

https://chanzuckerbergteam.slack.com/archives/C01C310V70X/p1616795763052000?thread_ts=1616795713.051900&cid=C01C310V70X